src/app/children/health-checkup/HealthCheck.ts

```diff
@@ -15,8 +15,8 @@
  *     along with ndb-core.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { Entity } from '../entity/entity';
-import {Child} from '../children/child';
+import { Entity } from '../../entity/entity';
+import {Child} from '../child';
 
 export class HealthCheck extends Entity{
 
```

Comment:
better use `/**` style code comments to describe the attributes below. With that we can possibly generate some documentation automatically in the future (e.g. https://typedoc.org/guides/doccomments/).

Usually the IDE even automatically creates a comment template for you when you type `/**` in the line above a statement and press enter.

Reply:
I get what you're saying and implemented most of the comments using /**. Anyway, I dont see the connection from the commenting style to the code snipped you used...